Initialize App_CI.yml : with @bassamEldsouki

ediz-genc opened the pull request:
Subject: Initialize App CI
type: User Story
Problem: App CI
git tags: User story, Sprint1, High

Resolves user story #20

ediz-genc committed:
Initialize App_CI.yml : with @bassamEldsouki

Subject: Initialize App CI
type: User Story
Problem: App CI
git tags: User story, Sprint1, High

Resolves user story #20

bassamEldesouki commented:
An App CI error because it can't find the directory we specified in the App_CI.yml file. This will be fixed after merging the App Android file to master.

bassamEldesouki commented on .github/workflows/App_CI.yml:
```diff
@@ -0,0 +1,26 @@
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: |
+        cd app
```
Needs to be changed to the right directory!

bassamEldesouki reviewed (changes requested):
It can't find the directory, Please write the right one!

ediz-genc commented:
> It can't find the directory, Please write the right one!

I am aware : ) Waiting for the colleague to change the directory of the app so we can add the right directory.

ediz-genc committed:
Update App_CI.yml

ediz-genc commented on .github/workflows/App_CI.yml:
```diff
@@ -0,0 +1,26 @@
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: |
+        cd tree/UI/UI/app
```
The directory for the app has been changed.

ediz-genc committed:
New directory added

Resolves #20

ediz-genc committed:
New directory added upon error in building

Resolves #20

platisd commented:
Read [how to create a commit with multiple authors](https://docs.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors). No need to tag like that on the pull request title (it doesn't provide more visibility)

ediz-genc committed:
New directory added

Because of tiny conflicts during merging to master, directory has to be changed again

Resolves #20

ediz-genc commented on .github/workflows/App_CI.yml:
```diff
@@ -0,0 +1,26 @@
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: |
+        cd UI
```
"Cd UI" added

bassamEldesouki committed:
Andriod CI workflow

Updating the CI workflow for the App

bassamEldesouki reviewed (approved):
Fixed error  " can't find the directory of gradlew"

ChrisAxt committed:
Updating app CI

Try to fix android app CI

bassamEldesouki committed:
Another try to fix the App CI

Fixing (error: gradlew: does not exist and --remove not passed)

bassamEldesouki reviewed (approved):
It could be a solution!

bassamEldesouki commented:
This issue has been fixed with issue #38
